hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/api/records/Resource.java

```diff
@@ -464,7 +464,7 @@ public int compareTo(Resource other) {
 
   @Override
   public String toString() {
-    return getFormattedString(String.valueOf(getMemorySize()));
+    return getFormattedString();
```

Comment:
Isn't it dangerous to do it for everything and not just for the UI?

Reply:
Thanks for your suggestion, I will modify the code of the NodeLabel page.